kube_obj_parser.py

```diff
@@ -9,6 +9,9 @@
 from sdcclient import SdcClient
 import time
 from time import gmtime, strftime
+# fix the 'InsecureRequestWarning' error
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
```

Comment:
Would it make sense to move the lines to the main Python script (https://github.com/draios/sysdig-kube-watcher/blob/master/kubewatcher.py)?

Reply:
@davideschiera Thanks for the comments. I have moved these lines to the `kubewattcher.py` script.